### ISSUE TITLE
Web3 1.0 `send`, `sendAsync` and `sendSync`

### ISSUE THREAD

**Issue by Dan-Nolan** [ethereum/web3.js#1038]
I'm trying to use web3.js 1.0 in conjunction with ethereumjs-testrpc. It seems like all the methods in web3.js are now using `send` where they were previously using `sendAsync` and `sendSync` where they were previously using `sync`. This is causing an issue when I try to run asynchronous methods with the testrpc provider:

```
var Web3 = require('web3');
var TestRPC = require('ethereumjs-testrpc');
const web3 = new Web3(TestRPC.provider());
web3.eth.getAccounts().then(console.log);
```

This code will result in `Synchronous requests are not supported.` because `send` is being called on the testrpc provider. 

Did I set this up wrong or does one of these two libraries need to conform to the other's standards?

**Comment by Dan-Nolan** [comment]
Ah, it looks like this would be fixed by #1018

**Comment by Dan-Nolan** [comment]
Fixed by #1018.
